packages/ra-ui-materialui/src/form/SimpleFormIterator.tsx

```diff
@@ -319,6 +319,7 @@ export interface SimpleFormIteratorProps
     extends Partial<Omit<FieldArrayRenderProps<any, HTMLElement>, 'meta'>> {
     addButton?: ReactElement;
     basePath?: string;
+    classes?: Partial<ReturnType<typeof useStyles>>;
```

Comment:
Would you mind using the `ClassesOverride` type? It can be imported from the `types.ts` file

```tsx
classes: ClassesOverride<typeof useStyles>;
```

Reply:
Sure no problem.

Reply:
Done.